tests/test_SALSubsystems.py

```diff
@@ -10,7 +10,10 @@
 
 INDEX_ENUM_CHECK = re.compile(r"[^,= \w]+")
 
-VALID_LANGUAGES = {"CPP", "Java", "LabVIEW", "IDL", "Python"}
+# These are the "languages" we need to build libraries for. Note we removed
+# support for python runtime (previously SALPY) and the python/salobj
+# components will either use IDL (for DDS) or xml directly (for kafka).
```

Comment:
I think you should clarify if the runtime would be blank if kafka or remain IDL for both.

Reply:
that is a good point.. however, I don't think we are ready to make that call

Reply:
Maybe remove the "or xml directly (for kafka)" since that could be potentially confusing for the context of choosing a runtime language.

Reply:
ok